sda-commons-server-s3/src/main/java/org/sdase/commons/server/s3/health/S3HealthCheck.java

```diff
@@ -0,0 +1,32 @@
+package org.sdase.commons.server.s3.health;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.codahale.metrics.health.HealthCheck;
+import com.codahale.metrics.health.annotation.Async;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Async(period = 30, scheduleType = Async.ScheduleType.FIXED_DELAY)
+public class S3HealthCheck extends HealthCheck {
+
+  private static final Logger LOG = LoggerFactory.getLogger(S3HealthCheck.class);
+
+  private final AmazonS3 s3Client;
+  private final String bucketName;
+
+  public S3HealthCheck(AmazonS3 s3Client, String bucketName) {
+    this.s3Client = s3Client;
+    this.bucketName = bucketName;
+  }
+
+  @Override
+  protected Result check() {
+    try {
+      s3Client.doesBucketExistV2(bucketName);
```

Comment:
Should we evaluate the return value? I.e. return `unhealthy` if the bucket does not exist?